src/karmen_frontend/src/routes/printers/printer-detail.js

```diff
@@ -166,17 +166,33 @@ const PrinterDetail = ({
                     Disconnect
                   </button>
                 ))}
+              {printer.client.pill_info ? (
```

Comment:
No tests present, please add one, it's really simple. I bet you can make it on your own @tibor-kulcar.

Reply:
Also, this can be simplified further:

```jsx
{printer.client.pill_info && printer.client.pill_info.update_available && (<div>your content</div>)}
```

No need for the other conditional branch.

Reply:
247da114d0cb0121f6a38543ab3f6ec7bd540bf1